src/Equinox.MessageDb/Tracing.fs

```diff
@@ -3,43 +3,53 @@ module internal Equinox.MessageDb.Tracing
 
 
 open System.Diagnostics
+open Equinox.Core.Tracing
 
 let source = new ActivitySource("Equinox.MessageDb")
 
 let addOpAttempt (attempt: int) (act: Activity) = if act <> null then act.AddTag("eqx.op_attempt", attempt) |> ignore
```

Comment:
see refactor in https://github.com/jet/equinox/pull/355

Reply:
:+1: moved it into here now